CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # minimal-cipher ChangeLog
 
+### 1.4.1 - TBD
```

Comment:
Seems to be missing mention of the changes/fix in the test suite.

Reply:
Added here: https://github.com/digitalbazaar/minimal-cipher/pull/29/commits/5583046fb336464f46cb5ce7b5dd14de660d52ad